src/layouts/Authenticated.vue

```diff
@@ -165,6 +165,12 @@ export default {
           to: '/admin',
           disabled: !this.currentUser.isAdmin,
         },
+        {
+          key: 'reports',
+          icon: 'history',
+          text: this.$t('dashboard.reports'),
```

Comment:
If you remove the `text` element here, the `NavButton` component will use the `key` item and fill in the correct localization